nncf/api/compression.py

```diff
@@ -0,0 +1,316 @@
+#
+#  Copyright (c) 2019-2020 Intel Corporation
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+"""
+@package docstring
+This package defines the API for the NNCF compression methods so that the user could
+extend the existing algorithms.
+"""
+from typing import Any, Dict, Optional, TypeVar
+
+from nncf import NNCFConfig
+from nncf.common.graph.model_transformer import ModelTransformer
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.common.utils.ordered_enum import OrderedEnum
+
+DOMAIN_CUSTOM_OPS_NAME = "org.openvinotoolkit"
```

Comment:
I am not sure about the current phase of unification but at some point `DOMAIN_CUSTOM_OPS_NAME ` should be moved to PyTorch-specific code.

Reply:
Rolled back the changes.